prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/model/PMMLModel.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.prinz.pmml.model
+
+import org.jpmml.evaluator.Evaluator
+import org.jpmml.evaluator.LoadingModelEvaluatorBuilder
+import java.io.File
+
+class PMMLModel {
+  //TODO we need evaluator in PMMLModel to get the signature
```

Comment:
let's keep in private field some input stream instead of using File class here in order to make it clear that we would not work with Files but rather with inputstreams